website/pages/en/playground.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+const CompLibrary = require('../../core/CompLibrary.js');
+const React = require('react');
+const Container = CompLibrary.Container;
+
+class Playground extends React.Component {
```

Comment:
Ah, now I understand why it would be nice to migrate to Docusaurus 2: to get support to React Hooks and not have to rely on class components!

Appreciate the code here follows existing code base style and practices, even if they are a little old-fashioned!